plugins/processors/ifname/ifname.go

```diff
@@ -390,13 +391,13 @@ func buildMap(gs snmp.GosnmpWrapper, tab *si.Table, column string) (nameMap, err
 		if err != nil {
 			return nil, fmt.Errorf("index tag isn't a uint")
 		}
-		nameIf, ok := v.Fields[column]
+		nameIf, ok := v.Fields["ifName"]
 		if !ok {
-			return nil, fmt.Errorf("field %s is missing", column)
+			return nil, errors.New("ifName field is missing")
```

Comment:
@srebhan I was wondering if this is the correct way to do it, if there are no arguments to the message.

Reply:
Yes that's the correct way, using `errors.New()` if you don't have arguments.